### PR TITLE
added /bitrate

### PR DESCRIPTION
**Pull request by ghost**
When I tried to fix my Git tree after a rebasing mistake, it wiped out the commit that the PR was referencing, which auto-closed it.

The code as it sits now should work perfectly fine (I updated help_onKey to reflect the new height).

### CODE REVIEW

File: src/audio_call.c [hunk 1]
```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
...
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = atoi(argv[1]);
+
+    TOXAV_ERR_BIT_RATE_SET error;
+    audio_bit_rate_status_cb(CallControl.av, CallControl.calls[self->num].out_idx, bitrate, -1, &error);
```
Comment (JFreegman) [inline src/audio_call.c]: This fix has regressed (second arg should be self->num)

File: src/audio_call.c [hunk 2]
```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
...
+
+    if (error != TOXAV_ERR_BIT_RATE_SET_OK) {
+        switch (error) {
+            case TOXAV_ERR_BIT_RATE_SET_SYNC:
+                error_str = "Syncronization error occured";
+                break;
+
+            case TOXAV_ERR_BIT_RATE_SET_INVALID_AUDIO_BIT_RATE:
+                error_str = "Invalid audio bit rate value";
```
Comment (JFreegman) [inline src/audio_call.c]: Perhaps add the range of allowed values in the error message

File: src/audio_call.c [hunk 3]
```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
...
+        error_str = "Must have value!";
+        goto on_error;
+    }
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = atoi(argv[1]);
```
Comment (JFreegman) [inline src/audio_call.c]: I missed this before but you shouldn't be using atoi(). Use strtol() instead, and be sure to error check it. There are lots of examples you can copy in *_commands.c

**Review by JFreegman — CHANGES_REQUESTED** [review]

File: src/audio_call.c (hunk 3 above)
Comment (ghost) [reply src/audio_call.c]: Error checking is done through the switch at the bottom (bounds checking between 6 and 510 triggers). Should I add a second check inside of the main function that also prints "Invalid audio bit rate value"?

File: src/audio_call.c (hunk 1 above)
Comment (ghost) [reply src/audio_call.c]: fixed that

File: src/audio_call.c (hunk 2 above)
Comment (ghost) [reply src/audio_call.c]: done

File: src/audio_call.c (hunk 3 above)
Comment (JFreegman) [reply src/audio_call.c]: Yeah on second thought forget about error checking strtol(). It's normally a good idea, but in this case it's redundant.

File: src/audio_call.c [hunk 4]
```diff
@@ -834,6 +835,55 @@ void cmd_sense(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)[M
...
+        error_str = "Must have value!";
+        goto on_error;
+    }
+
+    if ( self->is_call == false ) {
+        error_str = "Must be in a call";
+        goto on_error;
+    }
+
+    const uint32_t bitrate = strtol(argv[1]);
```
Comment (JFreegman) [inline src/audio_call.c]: This is giving a compile error. You need to supply some additional arguments.

**Comment by ghost** [comment]
Fixed that compile error, forgot to check over my work for a second

**Pull request merged**